### PR TITLE
Cleanup old metadata stuff

### PR DESCRIPTION
**Pull request by LipuFei**
Clean up some old metadata stuff.

### CODE REVIEW

**Comment by LipuFei** [comment]
retest this please

**Comment by LipuFei** [comment]
@whirm should be ready

**Comment by whirm** [comment]
Looks fine to me

**Comment by LipuFei** [comment]
@whirm @NielsZeilemaker ready

**Comment by whirm** [comment]
@LipuFei let's wait to see if @NielsZeilemaker comes with an obscure reason to keep the types table on the DB :D

**Comment by LipuFei** [comment]
@whirm @NielsZeilemaker I did a rebase and made one small fix.

File: Tribler/Main/vwxGUI/SearchGridManager.py [hunk 1]
```diff
@@ -534,12 +533,17 @@ def refreshGrid(self, remote=False):
 
     @call_on_reactor_thread
     def modifyTorrent(self, torrent, modifications):
+        # TODO: change this
```
Comment (NielsZeilemaker) [inline Tribler/Main/vwxGUI/SearchGridManager.py]: This one can probably go, as we only need to be able to modify channeltorrents.

File: Tribler/Test/test_metadata_community.py [hunk 2]
```diff
@@ -1,127 +0,0 @@
...
-
-from Tribler.Test.test_as_server import TestGuiAsServer, TESTS_DATA_DIR
-
-from Tribler.Core.TorrentDef import TorrentDef
-from Tribler.Core.simpledefs import dlstatus_strings
-
-DEBUG = True
-
-
-class TestMetadataCommunity(TestGuiAsServer):
```
Comment (NielsZeilemaker) [inline Tribler/Test/test_metadata_community.py]: I would keep this one, and see if you could merge it with the community stuffs. A download from an open-channel should trigger the same behaviour right?

File: Tribler/community/channel/community.py [hunk 3]
```diff
@@ -1187,49 +1178,49 @@ def _get_torrent_id_from_message(self, dispersy_id):
 
         return self._channelcast_db._db.fetchone(u"SELECT id FROM _ChannelTorrents WHERE dispersy_id = ?", (dispersy_id,))
 
-    def _get_latest_modification_from_channel_id(self, type_id):
-        assert isinstance(type_id, (int, long)), "type_id type is '%s'" % type(type_id)
+    def _get_latest_modification_from_channel_id(self, type_name):
+        assert type_name in METADATA_TYPES, "invalid type_name: %s" % repr(type_name)
```
Comment (NielsZeilemaker) [inline Tribler/community/channel/community.py]: I wouldn't check for the type, because peers are free to "choose" new types.

**Comment by NielsZeilemaker** [comment]
@whirm no problems found mr spanish guy

File: Tribler/Test/test_metadata_community.py (hunk 2 above)
Comment (LipuFei) [inline Tribler/Test/test_metadata_community.py]: OK, I will keep this one.

File: Tribler/Main/vwxGUI/SearchGridManager.py (hunk 1 above)
Comment (LipuFei) [inline Tribler/Main/vwxGUI/SearchGridManager.py]: This one is gone in another branch of mine, which is a successor of this one. It's getting a little huge, so I split it into several small PRs, doing the metadata thing bit by bit.

Comment (NielsZeilemaker) [inline Tribler/Main/vwxGUI/SearchGridManager.py]: That is a great idea

**Comment by LipuFei** [comment]
@whirm @NielsZeilemaker ready when it passes

**Comment by LipuFei** [comment]
retest this please

**Comment by LipuFei** [comment]
@whirm ready

**Pull request merged**